src/main/resources/fabric.mod.json

```diff
@@ -19,7 +19,6 @@
   "contributors": [
     {
       "name": "adammakesfilm",
-      "comment": "icon author",
```

Comment:
<html>
<body>
<!--StartFragment-->

[19:52:42] [ForkJoinPool-1-worker-1/WARN]: The mod "carpet-shadow" contains invalid entries in its mod json:
 - Invalid entry in person "comment" at line 22 column 16

<!--EndFragment-->
</body>
</html>